src/ipc/ipc4/handler.c

```diff
@@ -811,7 +811,7 @@ static int ipc4_module_process_dx(union ipc4_message_header *ipc4)
 		return IPC4_INVALID_RESOURCE_ID;
 	}
 
-	dcache_invalidate_region((void *)(MAILBOX_HOSTBOX_BASE), sizeof(dx_info));
+	dcache_invalidate_region((__sparse_force void __sparse_cache *)MAILBOX_HOSTBOX_BASE, sizeof(dx_info));
```

Comment:
btw, can we do something like Linux here and use a shorter attribute here i.e.
```c
#define __iomem (__sparse_force void __sparse_cache *)
```
?

Reply:
I'd be glad to have a shorter version of them, maybe even defining
```
#defing __cache __sparse_cache
```
although that would be kind of impolite to Zephyr... But I don't think it's a good idea to put both together. In most cases we only need `__sparse_cache` and only sometimes we need to add `__sparse_force` and I think those two should be easily distinguishable. And those pointers aren't always `void *` in fact more often than not they aren't. E.g. `struct comp_buffer __sparse_cache *` etc. And Linux `__iomem` is **only** an address space attribute, no pointer type and no `__attribute__((force))`

Reply:
ok, go you - this is Zephyr convention.